index.js

```diff
@@ -120,3 +167,15 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+
+
+
+function multiplyByTwo (number= ""){
```

Comment:
Providing a default value in the parameter is a nice tool, but definitely not required. Here you wouldn't want to assign a default value that doesn't match up with the type you expect for the variable. So if you expect the parameter `number` to be of type `number` it would be better to provide a value of type number like 0 as the default value instead of the empty string which is of type string.

So this could just be 
```
function multiplyByTwo (number){
    return number * 2;
}
```

or if you wanted to provide a default, you could do 
```
function multiplyByTwo (number = 0){
    return number * 2;
}
```
invoking the above function with a default without passing in an argument like this would return 0
`multiplyByTwo ()` // returns 0